bin/cide.conf.example

```diff
@@ -0,0 +1,5 @@
+[DEFAULT]
+server = ./configs/server.conf
+welcomeController = ./configs/welcomeController.conf
+ideController = ./configs/ideController.conf
```

Comment:
If I quote them, the quotes get escaped and the os.path.join() returns something like :
`bin/'./configs/server.conf'`.
It bugged me, but I didn't find a way to fix it.